KarhooUISDK/Controllers/NavigationController.swift

```diff
@@ -11,10 +11,51 @@ import UIKit
 
 final class NavigationController: UINavigationController {
 
+    enum Style {
+        /// Style uses primary color as bar tint color
+        case primary
+        /// Style uses background color as bar tint color
+        case secondary
+
+        var backgroundColor: UIColor {
+            switch self {
+            case .primary:
+                return KarhooUI.colors.primary
+            case .secondary:
+                return KarhooUI.colors.white
```

Comment:
We'll need to rename white and black as the names are confusing when in dark mode. Maybe `defaultBackground`? Which would be white in light mode and black in dark mode? Not for now, but it's a discussion we need to have.

Reply:
I think the idea is so we have `white` that is acctualy always white, no matter on light/dark mode. In this case the background color needs to be changed to `background1` or any else, but we need to update the design to do it properly.